src/main/webapp/app/shared/date-time-picker/date-time-picker.component.ts

```diff
@@ -26,7 +26,9 @@ export class FormDateTimePickerComponent implements ControlValueAccessor {
     @Input() min: dayjs.Dayjs; // Dates before this date are not selectable.
     @Input() max: dayjs.Dayjs; // Dates after this date are not selectable.
     @Input() shouldDisplayTimeZoneWarning = true; // Displays a warning that the current time zone might differ from the participants'.
-    @Output() valueChange = new EventEmitter();
+    @Output() valueChange: EventEmitter<boolean> = new EventEmitter();
```

Comment:
Please document this output and mention that this only returns whether the value is valid, and not the actual value. For me, from the name `valueChange` I'd expect to get the newly selected value and not a boolean, so maybe even rename this attribute?